### PR TITLE
Invite links update

### PR DESCRIPTION
**Pull request by sreynen**
For https://github.com/MoveOnOrg/Spoke/issues/86, this adds a hash column to the invite schema, populates it with UUID, and uses that instead of ID, so invite links are less guessable.

### CODE REVIEW

File: src/server/api/schema.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { log } from '../../lib'
```
Comment (shakalee14) [inline src/server/api/schema.js]: Why are we importing log here? Are we using it/planning to use it in this file?

**Review by shakalee14 — APPROVED** [review]

**Comment by shakalee14** [comment]
I think we need to wrap the inviteInput in an array because of knex psql? Not sure if this is the same issue, but in order to do an [insert statement and return the inserted id for knex and (specifically) psql, what you want to return needs to be wrapped in an array](http://knexjs.org/#Builder-insert). But this comment could be completely un related.

**Review by shakalee14 — COMMENTED** [review]
When testing on production, I'm getting a GraphQL error when trying to create an organization. After you created an invite connected to a hash, were you able to create an organization?

**Comment by sreynen** [comment]
Org creation should be fixed now. In my testing, the org I created doesn't get attached to my account, but I confirmed the org was created in the database, and I believe that also happens in master, so assuming it's not related to invite changes.

**Pull request merged**